api/src/pcapi/core/bookings/models.py

```diff
@@ -270,31 +233,19 @@ def is_cancelled(cls) -> bool:  # pylint: disable=no-self-argument
 
     @property
     def firstName(self) -> str | None:
-        if self.individualBooking is not None:
-            return self.individualBooking.user.firstName
-
-        raise ValueError(f"Booking {self.id} has no individual booking.")
+        return self.user.firstName
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏On peut même retirer ces properties
Ce sera plus clair d'avoir `booking.user.name` que `booking.name` qui ne veut pas dire grand chose

Reply:
je prefererai modifier le moins de choses possible, la pr est deja assez grosse.

Reply:
okay :+1: 
On peut ajouter un `# TODO` ou un ticket pour ça ?